src/System.IO.FileSystem/tests/System.IO.FileSystem.Tests.csproj

```diff
@@ -107,9 +107,6 @@
     <Compile Include="PortedCommon\IOServices.cs" />
     <Compile Include="PortedCommon\PathInfo.cs" />
     <Compile Include="PortedCommon\ReparsePointUtilities.cs" />
-    <Compile Include="PortedCommon\TemporaryDirectory.cs" />
```

Comment:
I didn't because I thought they were used by other projects (I saw Common and moved on).  Apparently not.  Will remove.